datastore/db/bytebase/FertiScan/new_inspection_function.sql

```diff
@@ -300,12 +300,13 @@ BEGIN
 
 -- INSPECTION
     INSERT INTO "fertiscan_0.0.12".inspection (
-        inspector_id, label_info_id, sample_id, picture_set_id
+        inspector_id, label_info_id, sample_id, picture_set_id, original_dataset
```

Comment:
`original_dataset` could easily be part of the OLAP layer. There is no use for it in the operational aspects. I suggest it be another dimension.